Resources/Private/Templates/Event/BeList.html

```diff
@@ -1,14 +0,0 @@
-<f:layout name="BeModule" />
```

Comment:
This is a bit funny template, but it is necessary. If you don't have it, you get the following error message if you call no page or a page tree without the slub_events template.

![SLUB Webseite  TYPO3 CMS 9 5 27  (1)](https://user-images.githubusercontent.com/3354500/121369028-cf1d7600-c93b-11eb-84ba-305ac32ff3c5.png)

With this template, it looks like this:

![SLUB Webseite  TYPO3 CMS 9 5 27 ](https://user-images.githubusercontent.com/3354500/121368832-a5644f00-c93b-11eb-93b2-0afba91422a4.png)

Reply:
One of the next things I want to do is, to migrate the template stuff beneath _Resources/Private/Backend_ into

- _Resources/Private/Templates_
- _Resources/Private/Partials_ 
- _Resources/Private/Layouts_ 

In this case the conventions, related to the controller structure, are fulfilled and the beList template will be found.

Is there anything that speaks against?

Reply:
I'm not sure, that this is working. I would keep this PR on hold for now.

`news` as reference keeps the backend template separate, too. I think this structure was made by the extension manager in former times. Of course this may be not necessary today.

Have a try and come back to this PR.